helm/chart/router/templates/deployment.yaml

```diff
@@ -126,9 +126,9 @@ spec:
               {{- include "common.tplvalues.render" (dict "value" .Values.extraVolumeMounts "context" $) | nindent 12 }}
             {{- end }}
           {{- end }}
-          {{- if .Values.extraContainers }}
-          {{- tpl (toYaml .Values.extraContainers) $ | nindent 10 }}
```

Comment:
Worth noting I don't understand the before nor the after, merely copied from context.